doc/api/assert.md

```diff
@@ -209,8 +209,8 @@ changes:
   - version: v9.0.0
     pr-url: https://github.com/nodejs/node/pull/15036
     description: NaN is now compared using the
-                 [SameValueZero](https://tc39.github.io/ecma262/#sec-samevaluezero)
-                 comparison.
+              [SameValueZero](https://tc39.github.io/ecma262/#sec-samevaluezero)
```

Comment:
The fact that a change like this has to be made is frustrating and not intuitive.

Reply:
The issue here is that another linting rule does not check comments and using `[SameValueZero][]` is not possible. It would of course be great if that would be fixed instead.